package.json

```diff
@@ -46,8 +46,13 @@
   },
   "devDependencies": {
     "@ethersproject/cli": "5.5.0",
+    "@types/chai": "^4",
```

Comment:
the newly added `dependancies` and `devDependancies` should be moved in `hoprd/package.json`